final_task/rss_reader/CSVEntities.py

```diff
@@ -0,0 +1,50 @@
+import csv
+from datetime import date
+from dateutil.parser import parse
+
+import ClassNews
+
+FIELDNAMES = ['date', 'title', 'link', 'article', 'links']
+
+
+def csv_to_python(articles_list, csv_file):
+    """This function inserts news to the source csv file that has never been seen in it."""
+    articles_list_from_csv = []
+    with open(csv_file, "r") as file:
+        reader = csv.DictReader(file, FIELDNAMES, delimiter='\t')
+        for items in reader:
+            r = ClassNews.Article(dict(items))
+            articles_list_from_csv.append(r)
+
+    union_list = articles_list_from_csv[:]
+    for item in articles_list:
+        if item not in articles_list_from_csv:
+            union_list.append(item)
+
+    with open(csv_file, "w") as file:
+        writer = csv.DictWriter(file, fieldnames=FIELDNAMES, delimiter='\t')
+        for item in union_list:
+            writer.writerow(item.__dict__)
```

Comment:
Checkout `asdict` method from `dataclasses` lib

Reply:
Okay, thank you, I'll fix it soon